pyvista/core/dataset.py

```diff
@@ -103,32 +103,100 @@ def __getattr__(self, item) -> Any:
 
     @property
     def active_scalars_info(self) -> ActiveArrayInfo:
-        """Return the active scalar's field and name: [field, name]."""
+        """Return the active scalar's field and name.
+
+        Field is the data association (e.g. point, cell, or field),
+        and name of the scalars.
+
+        Examples
+        --------
+        Create a mesh, add scalars to the mesh, and return the active
+        scalars info.  Note how when the scalars are added, they
+        automatically become the active scalars.
+
+        >>> import pyvista
+        >>> mesh = pyvista.Sphere()
+        >>> mesh['Z Height'] = mesh.points[:, 2]
+        >>> mesh.active_scalars_info
+        ActiveArrayInfo(association=<FieldAssociation.POINT: 0>, name='Z Height')
+
+        Notes
+        -----
+        If both cell and point scalars are present and neither have
+        been set active within at the dataset level, point scalars
+        will be made active.
```

Comment:
Is this still true? Seems like this was removed.

Reply:
This is still true.

Reply:
Then I misunderstand what that says. Because doing
```py
import pyvista as pv
mesh = pv.Sphere()
mesh.clear_arrays()
mesh['p'] = range(mesh.n_points)
mesh['c'] = range(mesh.n_cells)
mesh.set_active_scalars(None)
print(mesh.active_scalars is None)  # True
print(mesh.active_scalars_info)  # ActiveArrayInfo(association=<FieldAssociation.POINT: 0>, name=None)
print(mesh.active_scalars is None)  # True
```
I'd have expected the last line to be `False`: there are both cell and point scalars present, and neither are active. What is the missing bit?

Reply:
You didn't misunderstand, it looks like there's a bug...

Reply:
Actually, issue is `mesh.set_active_scalars(None)` disables all active scalars.

```py
import pyvista as pv
mesh = pv.Sphere()
mesh.clear_data()
mesh.point_arrays.set_array(range(mesh.n_points), 'p')
mesh['c'] = range(mesh.n_cells)
mesh.point_arrays.clear()
print(mesh.active_scalars is None)  # True
print(mesh.active_scalars_info)  # ActiveArrayInfo(association=<FieldAssociation.POINT: 0>, name=None)
print(mesh.active_scalars is None) 
```

```
False
ActiveArrayInfo(association=<FieldAssociation.CELL: 1>, name='c')
False
```

Reply:
I guess I still don't get it @akaszynski. `mesh.set_active_scalars(None)` disables all active scalars -- so the quoted docs explaining what happens when no point/cell scalars are set as active should exactly apply? There's probably something basic I'm missing here, and it's OK if we leave it at that. I just wanted to raise this in case this was a bug rather than confusion on my part.